src/server_manager/ui_components/outline-server-settings.html

```diff
@@ -398,7 +397,13 @@ <h3>[[localize('settings-metrics-header')]]</h3>
         }
       },
       _metricsEnabledChanged: function(newMetricsEnabled, oldMetricsEnabled) {
-        if (oldMetricsEnabled === undefined || newMetricsEnabled === undefined) {
+        // metricsEnabled is initialized to null to avoid firing the first time it is set.
```

Comment:
There's an issue with the design here. We are triggering events even when there's no user action.

It will be cleaner if you attach a listener for the 'change' event. That's only triggered by the user interaction. Then you can delete the observer.

The html will look like:
```html
 <paper-checkbox checked="{{metricsEnabled}}" on-change="handleMetricsChange"></paper-checkbox>
```

This way setting the property won't trigger any events.